### PR TITLE
Splash

### PR DESCRIPTION
**Pull request by pazdera**
We only need the graphic.

cc @alex5imon @TommySaehl @Ealdwulf

### CODE REVIEW

**Comment by TommySaehl** [comment]
youtube-splash

**Comment by alex5imon** [comment]
@pazdera graphic is here! Thanks @TommySaehl

File: debian/control [hunk 1]
```diff
@@ -10,7 +10,7 @@ Architecture: all
 Conflicts: make-video, youtube-dl, kano-youtube
 Provides: make-video, youtube-dl, kano-youtube
 Replaces: make-video, youtube-dl, kano-youtube
-Depends: ${misc:Depends}, kano-toolset (>=1.1-7), python,
+Depends: ${misc:Depends}, kano-toolset (>=1.3-8), python,
          fonts-takao, kano-fonts (>= 2.0-1)
```
Comment (alex5imon) [inline debian/control]: I think it needs to depend on kano-video-files (>=1.1-2) as well

**Comment by pazdera** [comment]
@TommySaehl Cheers man!

@alex5imon will add that in a separate PR.

**Pull request merged**